### PR TITLE
Rating system, back buttons, code enhancements and refactoring

### PR DESCRIPTION
**Pull request by MartinCepulionis**
- Added UserEditingService and moved all of the password, email, university, status change code there.
- Added rating system (from Very Bad to Very Good) to already implemented review system.
- Enhanced settings tab. Added MessageBox functionality and field clearing.
- Added BACK buttons in all of the forms.
- Fixed closing current forms and opening new ones (removed hiding forms and not closing them).

### CODE REVIEW

**Comment by MartinCepulionis** [comment]
Also I disabled MAXIMIZE, MINIMZE buttons in all forms and made our forms Fixed so that user can't change their size.

File: University-advisor/Forms/MainForm.cs [hunk 1]
```diff
@@ -295,13 +148,32 @@ private void UniversitiesGrid_CellClick(object sender, DataGridViewCellEventArgs
         private void ReviewSubmit_Click(object sender, EventArgs e)
         {
             string review = universityReview.Text;
-            if(review.Length == 0)
+            int rating = 0;
+            if (veryBadButton.Checked)
```
Comment (avizen-j) [inline University-advisor/Forms/MainForm.cs:154]: It is not great when you mix if/else statement structure. Most of the code was written using structure as in 233-238 lines. So lets stick to it.

**Review by avizen-j — CHANGES_REQUESTED** [review]
Structure mixing.

File: University-advisor/Forms/MainForm.cs (hunk 1 above)
Comment (MartinCepulionis) [reply University-advisor/Forms/MainForm.cs:154]: Fixed.

**Review by avizen-j — APPROVED** [review]

**Pull request merged**